2015-11-02 12:21:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "ID_DONE" not found; SQL statement:
UPDATE todos SET id_done = NOT id_done WHERE ROWNUM = ? [42122-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.table.Table.getColumn(Table.java:660)
	at org.h2.command.Parser.readTableColumn(Parser.java:700)
	at org.h2.command.Parser.parseUpdate(Parser.java:734)
	at org.h2.command.Parser.parsePrepared(Parser.java:454)
	at org.h2.command.Parser.parse(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:276)
	at org.h2.command.Parser.prepareCommand(Parser.java:241)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at ToDo.toggleTodo(ToDo.java:45)
	at ToDo.main(ToDo.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-11-02 12:21:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "ID_DONE" not found; SQL statement:
UPDATE todos SET id_done = NOT id_done WHERE ROWNUM = ? [42122-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.table.Table.getColumn(Table.java:660)
	at org.h2.command.Parser.readTableColumn(Parser.java:700)
	at org.h2.command.Parser.parseUpdate(Parser.java:734)
	at org.h2.command.Parser.parsePrepared(Parser.java:454)
	at org.h2.command.Parser.parse(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:276)
	at org.h2.command.Parser.prepareCommand(Parser.java:241)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at ToDo.toggleTodo(ToDo.java:45)
	at ToDo.main(ToDo.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-11-02 12:21:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "ID_DONE" not found; SQL statement:
UPDATE todos SET id_done = NOT id_done WHERE ROWNUM = ? [42122-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.table.Table.getColumn(Table.java:660)
	at org.h2.command.Parser.readTableColumn(Parser.java:700)
	at org.h2.command.Parser.parseUpdate(Parser.java:734)
	at org.h2.command.Parser.parsePrepared(Parser.java:454)
	at org.h2.command.Parser.parse(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:276)
	at org.h2.command.Parser.prepareCommand(Parser.java:241)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at ToDo.toggleTodo(ToDo.java:45)
	at ToDo.main(ToDo.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-11-02 12:23:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "ID_DONE" not found; SQL statement:
UPDATE todos SET id_done = NOT is_done WHERE ROWNUM = ? [42122-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.table.Table.getColumn(Table.java:660)
	at org.h2.command.Parser.readTableColumn(Parser.java:700)
	at org.h2.command.Parser.parseUpdate(Parser.java:734)
	at org.h2.command.Parser.parsePrepared(Parser.java:454)
	at org.h2.command.Parser.parse(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:276)
	at org.h2.command.Parser.prepareCommand(Parser.java:241)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at ToDo.toggleTodo(ToDo.java:45)
	at ToDo.main(ToDo.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
